lua_examples/lfs/HTTP_OTA.lua

```diff
@@ -0,0 +1,77 @@
+--
+-- If you have the LFS _init loaded then you invoke the provision by 
+-- executing LFS.HTTP_OTA('your server','directory','image name').  Note
+-- that is unencrypted and unsigned. But the loader does validate that
+-- the image file is a valid and complete LFS image before loading.
+--
+
+local host, dir, image = ...
+
+local doRequest, firstRec, subsRec, finalise
+local n, total, size = 0, 0
+  
+doRequest = function(sk,hostIP)
+  if hostIP then 
+    local con = net.createConnection(net.TCP,0)
```

Comment:
@TerryE I recently tried modifying your HTTP_OTA example using `tls.createConnection` instead, so updated LFS images could be retrieved from a secure server such as AWS s3. It didn't work, however -- the connection was established and got a success back from the server, but it seems that the `on("receive",..)` callback was never called subsequently with the body of the file to download. I would have expected the tls submodule to be equivalent to the net module in retrieving a file.

Am I missing something obvious?